go.mod

```diff
@@ -9,27 +9,27 @@ require (
 	github.com/google/go-cmp v0.5.6
 	github.com/jaypipes/ghw v0.9.0
 	github.com/jaypipes/pcidb v1.0.0
-	github.com/k8stopologyawareschedwg/deployer v0.7.3-0.20220913163638-4d990c6a4f14
+	github.com/k8stopologyawareschedwg/deployer v0.8.0
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.12
 	github.com/k8stopologyawareschedwg/podfingerprint v0.1.1
-	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.7.6
+	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.8.1
 	github.com/kubevirt/device-plugin-manager v1.19.4
 	github.com/mdomke/git-semver v1.0.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
 	github.com/openshift/api v0.0.0-20210924154557-a4f696157341
 	github.com/openshift/machine-config-operator v0.0.1-0.20211105081319-76d6155c1dab
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.0
 	google.golang.org/grpc v1.40.0
-	k8s.io/api v0.23.3
+	k8s.io/api v0.24.7
```

Comment:
What is reason behind not updating to the latest kube version i.e. 1.25.3 ?

Reply:
two reasons:
- minor: I want to have incremental steps and not just one big jump
- major: kubernetes 1.25+ pulls ginkgo v2, and we need to adapt our suite. Upgrading to 1.24 is cheaper and moves us in a better position for this much larger update.

Reply:
Ack, makes sense. Thanks for the explanation.